client/src/views/Graphs/components/DrilldownPanel/NodePane.vue

```diff
@@ -21,11 +21,9 @@
       <details v-if="data.in_degree" class="metadata" open>
         <summary>Incoming ({{ data.in_degree }})</summary>
```

Comment:
I think these counters are not reflecting the actual neighbors that we get from bgraph?